README.txt

```diff
@@ -12,8 +31,37 @@ Run the following commands to test reading and writing to the server. You can ad
 read read-test.txt localhost
 write write-test.txt localhost
 
+
+Test File Locations:
+=====================
 The locations for reading and writing are listed under the `resources` folder.
 - `resources/client` for files on the client
 - `resources/server` for files on the server
 
 You can place files there and trying reading to the client folder or writing to the server folder. 
+
+
+Diagrams:
+=====================
+The diagrams will be in /src/main/java/resources folder. They will contain
+- UCM for Read File transfer
+- UCM for Write File transfer
+- UML Diagram
+
+Project Structure
+=====================
+/src/main/java/
+    - FTPClient.java - The Java Client
+    - FTPServer.java - The Java server
+    - components/ErrorSimulator.java - The Error Simulator created by Client
+    - exceptions/ - projects specific exceptions for invalid packet and invalid commands
+    - formats/ - All the message types that are involved in the TFTP protocol
+    - logging/ - Simple logger
+    - parsing/ - FTPClient command line interface actions separated by command
+    - resources/client - The director where the client looks for reads/writes
+    - resources/server - The director where the server looks for reads/writes
+    - resources/diagrams - The Diagrams needed for iteration 1
```

Comment:
these three are at the project root. `/src/main/java/resources` contain the resource manager class